.travis.yml

```diff
@@ -51,6 +51,9 @@ install:
   - if [[ $MINIMAL_ENV == false ]] ; then
       DEPS="${DEPS} ${DEPS_OPTIONAL}";
     fi
+  - if [[ $ENV == pip ]] ; then
```

Comment:
Could you add some comments here explaining why this is needed?

Reply:
This was to check that the pillow version was the issue. pillow 7.1.1 have been released a few hours ago, once it is available on pypi, I will remove this.